frontend/src/components/assessments/student-experience/AssessmentRules.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
```

Comment:
can we change the assessmentMetadata.rules to not have newlines (since i don't think we would have that from the input). and then fix the code and reformat width